src/storagesync/azext_storagesync/_params.py

```diff
@@ -0,0 +1,159 @@
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('storagesync sync-group list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+
+    with self.argument_context('storagesync cloud-endpoint create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type)
+        c.argument('cloud_endpoint_name', cloud_endpoint_name_type, options_list=['--name', '-n'])
+        c.argument('storage_account_resource_id', storage_account_type)
+        c.argument('azure_file_share_name', azure_file_share_name_type)
+        c.argument('storage_account_tenant_id', storage_account_tenant_id_type)
+
+    with self.argument_context('storagesync cloud-endpoint delete') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type)
+        c.argument('cloud_endpoint_name', cloud_endpoint_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('storagesync cloud-endpoint show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type)
+        c.argument('cloud_endpoint_name', cloud_endpoint_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('storagesync cloud-endpoint list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type)
+
+    with self.argument_context('storagesync cloud-endpoint wait') as c:
+        c.argument('cloud_endpoint_name', cloud_endpoint_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('storagesync server-endpoint create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('storage_sync_service_name', storage_sync_service_name_type)
+        c.argument('sync_group_name', sync_group_name_type)
+        c.argument('server_endpoint_name', server_endpoint_name_type, options_list=['--name', '-n'])
+        c.argument('server_resource_id', server_resource_id_type)
+        c.argument('server_local_path', help='The local path of the registered server.')
+        c.argument('cloud_tiering', arg_type=get_enum_type(['on', 'off']), help='Cloud Tiering.')
+        c.argument('volume_free_space_percent',
```

Comment:
The following four parameters are too long. It's also a little bit confused to me. Is it able to add more help messages and examples?

Reply:
I'll add more help for them and add use these parameters in example

---
In reply to: [392869872](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392869872) [](ancestors = 392869872)